python-get-all-files-in-directory/create_large_dir.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from pathlib import Path
+
+
+NUM_JUNK_FILES_LARGE = 300
+NUM_JUNK_FILES_MEDIUM = 25
+NUM_JUNK_FILES_SMALL = 5
+
+
+@dataclass
+class Item:
+    name: str
+    children: list[Item] = None
+    junk_files: int = None
```

Comment:
Should we rename this to `num_junk_files` for explicitness and consistency?

Also, isn't `0` a better default value? If we change this, we should update the test later to be `if ... not item.junk_files` instead of `if ... item.junk_files is None`. Additionally, we don't need the `if item.junk_files` test since we'll only be looping over an empty range.